### PR TITLE
Facebook page like button Created

### PR DESCRIPTION
**Pull request by abhishhh1**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -98,6 +98,7 @@ <h1 class="hero-title">&nbsp;OPENCODE</h1>
  <hr class="card-underline">
  <div class="row quote">
    <p class="hero-quote">IIIT Allahabad's very own mini GSOC!</p>
+   <a href="https://www.facebook.com/opencodeiiita/" class="btn btn-primary" style="background-color: red;">Like Us on Facebook</a>
```
Comment (pradeepgangwar) [inline index.html]: This is not what we want. Read more about adding like button on facebook docs.

**Comment by pradeepgangwar** [comment]
Also refer to issue to you are working in the description.

File: index.html (hunk 1 above)
Comment (abhishhh1) [reply index.html]: I didn't get what to do then. I simply added a button to like facebook page of Opencode18. where is the docs ?

Comment (abhishhh1) [reply index.html]: Plz explain what to do?

Comment (pradeepgangwar) [reply index.html]: Read here https://developers.facebook.com/docs/plugins/like-button

**Comment by gauthamzz** [comment]
No need to request review. Merge if you think it’s right

File: index.html (hunk 1 above)
Comment (abhishhh1) [reply index.html]: I have updated code now, is this fine or needs more improvement

**Comment by pradeepgangwar** [comment]
Add a live link from your fork to see changes.

**Comment by pradeepgangwar** [comment]
Closing due to inactivity.

**Pull request closed without merging**